frontend/components/feedback-task/DatasetFilters.component.vue

```diff
@@ -58,12 +61,22 @@ export default {
     this.$root.$on("reset-search-filter", () => {
       this.searchInput = this.searchFromRoute;
     });
+    this.$root.$on("total-records", (newTotalRecords) => {
+      this.totalRecords = newTotalRecords;
+    });
 
     this.sortedFiltersValue = Object.values(this.filters).sort((a, b) =>
       a.order > b.order ? -1 : 1
     );
   },
   computed: {
+    additionalInfoForSearchComponent() {
+      if (isNil(this.totalRecords)) return null;
+
+      if ([0, 1].includes(this.totalRecords))
```

Comment:
Since we have already a page when a search returns no data, maybe we can skip the message when 0 records were found?

cc @nataliaElv @Amelie-V